public/static/locales/en/me.json

```diff
@@ -3,6 +3,7 @@
   "target": "Target",
   "share": "Share",
   "registerTrees": "Register Trees",
+  "manageProjects": "Manage Projects",
```

Comment:
We should try to avoid repeating too much duplicate content in translation resources (at least if the translation is within the same resource file). This is already existing as https://github.com/Plant-for-the-Planet-org/planet-webapp/blob/f358ad2e551ba81584c090d0ca88c105c59f0b09/public/static/locales/en/me.json#L11

Reply:
done

Reply:
Please do not just remove `settingManageProject` as it is of course used at https://github.com/Plant-for-the-Planet-org/planet-webapp/blob/b49b0a05d22130b0f9345244937383d84eeecc8c/src/features/user/UserProfile/components/SettingsModal.tsx#L98

Reply:
my bad, fixed.

Reply:
I did some more fixes and removed the now unused resource `manageProjects`.